lib/tokens/token.js

```diff
@@ -71,30 +77,20 @@ module.exports = function (log, random, P, hkdf, Bundle, error) {
         stack: err.stack
       })
     }
+    details.createdAt = optionallyOverrideCreatedAt(details.createdAt)
```

Comment:
I seem to remember, in a previous iteration of this change, that we decided it would be better to move this up to the route handler(s) so that it would more obviously apply only to whichever routes our tests need it on. I'm a bit hazy on the detail though and I can definitely see the case for encapsulating it here in the token object. Not sure if it really matters one way or the other, but mentioning in case you have any thoughts on it?